op-chain-ops/cmd/check-canyon/main.go

```diff
@@ -134,22 +134,22 @@ func ValidateReceipts(ctx Args, canyonActive bool) error {
 }
 
 func Validate1559Params(ctx Args, canyonActive bool) error {
```

Comment:
No need to skip this validation, it is L2 related (BaseFee will be included).

Reply:
done